### PR TITLE
Update screenshots

### PR DESCRIPTION
**Pull request by vchernin**
-Replace previous 3846x2166 ones with actual 3840x2160 ones
-Add 1280x720 screenshots

I couldn't take 1920x1080 screenshots natively without the interface looking way too big or too small. I think it'd be best to scale them down like in https://github.com/tenacityteam/assets/pull/5.

cc @TheEvilSkeleton @SFR-git 

P.S. also I got gpg working finally. For some reason the commit wasn't signed off at first, but I could fix that easily.

### CODE REVIEW

**Comment by TheEvilSkeleton** [comment]
> I couldn't take 1920x1080 screenshots natively without the interface looking way too big or too small. I think it'd be best to scale them down like in #5.

The issue is still open so @SFR-git could adapt it in the MR.

**Review by TheEvilSkeleton — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ The files within this repository are licensed under the Creative Commons Attribu
 
 `tenacity-icon-dark.svg`, `tenacity-icon-light.svg`, `tenacity-logo-dark.svg`, and `tenacity-logo-light.svg` were each modified or created by Emily Mabrey using material licensed to the Tenacity project by Rikard Jansson (under the `CC-BY-4.0`).
 
-`screenshots/tenacity-screenshot-light-3840x2160.png` and `screenshots/tenacity-screenshot-dark-3840x2160.png` are licensed to the Tenacity project under the CC BY 4.0 by Vincent Chernin.
\ No newline at end of file
+`screenshots/tenacity-screenshot-light-3840x2160.png`, `screenshots/tenacity-screenshot-dark-3840x2160.png`, `screenshots/tenacity-screenshot-light-1280x720.png` and `screenshots/tenacity-screenshot-dark-1280x720.png` are licensed to the Tenacity project under the CC BY 4.0 by Vincent Chernin.
```
Comment (SFR-git) [inline README.md:11]: Nitpick: Oxford comma?

**Review by SFR-git — COMMENTED** [review]
I'm going to rebase this in a bit.

**Comment by SFR-git** [comment]
Do you want to add a downscaled 1080p version as well? Or are we fine with 4K/720p?

**Comment by TheEvilSkeleton** [comment]
1080p would be nice to be honest.

**Comment by SFR-git** [comment]
I'm going to scale them down and finalize the commit, then

**Comment by SFR-git** [comment]
@TheEvilSkeleton: be advised that you will need to update the links in the manifest, as they're linked to a specific commit hash.

**Comment by TheEvilSkeleton** [comment]
@SFR-git I know.

**Review by SFR-git — APPROVED** [review]

**Review by TheEvilSkeleton — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ The files within this repository are licensed under the Creative Commons Attribu
 
 `tenacity-icon-dark.svg`, `tenacity-icon-light.svg`, `tenacity-logo-dark.svg`, and `tenacity-logo-light.svg` were each modified or created by Emily Mabrey using material licensed to the Tenacity project by Rikard Jansson (under the `CC-BY-4.0`).
 
-`screenshots/tenacity-screenshot-light-3840x2160.png` and `screenshots/tenacity-screenshot-dark-3840x2160.png` are licensed to the Tenacity project under the CC BY 4.0 by Vincent Chernin.
\ No newline at end of file
+All images under `screenshots/` are licensed to the Tenacity project under the CC BY 4.0 by Vincent Chernin.
```
Comment (TheEvilSkeleton) [inline README.md:11]: This should be at least inside LICENSE.md. It's fine if it's in the README but I'm pretty sure it's crucial for it to be in the license file. Here's an example from GitLab: https://gitlab.com/gitlab-org/gitlab/-/blob/master/LICENSE#L3-10

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
Actually, I just realized something.

File: README.md (hunk 2 above)
Comment (SFR-git) [reply README.md:11]: Well, since the entire repository is licensed under CC BY 4.0, that's irrelevant.

Comment (TheEvilSkeleton) [reply README.md:11]: Huh, you're right/

**Review by TheEvilSkeleton — APPROVED** [review]

**Comment by vchernin** [comment]
I just realized `tenacity-screenshot-light-1280x720.png` has a 2 in the top left since my display settings were left open… Whoops that’s definitely my bad.

I’ll fix that in a new PR I guess.

**Pull request closed without merging**